vdirsyncer/cli/__init__.py

```diff
@@ -38,7 +39,7 @@ def inner(*a, **kw):
 
 
 def validate_verbosity(ctx, param, value):
-    x = getattr(log.logging, value.upper(), None)
+    x = getattr(logging.getLogger, value.upper(), None)
```

Comment:
It looks like this function is never called though, so is it safe to delete it? Or did I miss something?